Libraries/Components/TextInput/TextInput.js

```diff
@@ -162,6 +162,21 @@ type IOSProps = $ReadOnly<{|
 |}>;
 
 type AndroidProps = $ReadOnly<{|
+  autoCompleteType?: ?(
+    | 'username'
+    | 'password'
+    | 'email'
+    | 'name'
+    | 'tel'
+    | 'street-address'
+    | 'postal-code'
+    | 'cc-number'
+    | 'cc-csc'
+    | 'cc-exp'
+    | 'cc-exp-month'
+    | 'cc-exp-year'
+    | 'off'
```

Comment:
Nit: You should alpha order this list. Unless you have a specific reason for it being in this order. I would alpha order all values and then add off at the end for clear readability.